### DISCUSSION TITLE
fabric哪来的1.14-？

### DISCUSSION THREAD

**Discussion by newbeflash** [Meloong-Git/PCL#5294] [讨论]
启动器搜Ornithe Standard Libraries模组，会发现这玩意要fabric的1.7.10版本的，怎么回事？给我震惊到了
![1](https://github.com/user-attachments/assets/b4e53100-216b-495d-bf7f-038e9e2b2462)
![2](https://github.com/user-attachments/assets/87f3709b-03ba-4994-9741-e3997dd21c04)

**Comment by newbeflash** [comment]
难道是开发者乱写的？

**Comment by 3gf8jv4dv** [comment]
你好，有的——Legacy Fabric。

它是原 Fabric 的一个 fork，专注于支持旧的 Minecraft 版本。

https://github.com/Legacy-Fabric/fabric
